sql/moz-fx-data-shared-prod/monitoring_derived/stable_table_column_counts_v1/script.sql

```diff
@@ -41,6 +41,12 @@ LOOP
     ''';
 END LOOP;
 
+-- Delete existing records for day.
+-- Prevents having duplicates when re-running query.
+DELETE FROM
+  monitoring_derived.stable_table_column_counts_v1
+WHERE
+  submission_date = DATE(@submission_date)
```

Comment:
One thing we should keep in mind, when backfilling we'll have the column numbers of the day when the backfill is executed for each `submission_date`. I think this is fine since we are generally interested in knowing whether we have reached the max columns. Keeping a record for historical dates is just a nice to have